More precise steps on keytab move and ownership change for realm support (#3643)
theforeman/foreman-documentation: guides/common/modules/con_configuring-project-to-manage-the-lifecycle-of-a-host-registered-to-a-freeipa-realm.adoc

----
# scp /root/freeipa.keytab root@_{smartproxy-example-com}_:/etc/foreman-proxy/freeipa.keytab
----

> . On the {ProjectServer}, if you are configuring it for realm support, move the `/root/freeipa.keytab` file to the `/etc/foreman-proxy` directory:

Reviewer: ```suggestion
. On your {ProjectServer}, move the `/root/freeipa.keytab` file to the `/etc/foreman-proxy` directory:
```

suggestion (non-blocking): Can we drop this because it's a step under the "for {FreeIPA} realm support" heading from line 38?

Another suggestion: Use "your {ProjectServer}" in favor of "the {ProjectServer}".

+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
# mv /root/freeipa.keytab /etc/foreman-proxy
----

> . On both the {ProjectServer} and the {SmartProxyServer}s, set ownership to the `foreman-proxy` user and group:

Reviewer: ```suggestion
. On your {ProjectServer} and {SmartProxyServer}s, set ownership to the `foreman-proxy` user and group:
```

suggestion (non-blocking): Reword slightly.
